Library/Homebrew/utils/gems.rb

```diff
@@ -296,6 +296,7 @@ def install_bundler_gems!(only_warn_on_failure: false, setup_path: true, groups:
       end
 
       bundle_installed = if bundle_install_required
+        Process::UID.change_privilege(Process.euid) if Process.euid != Process.uid
```

Comment:
Seems correct. I think you can make this temporary by doing something like
```ruby
begin
  original_uid = Process.uid
  Process::UID.change_privilege(Process.euid)
  ...
ensure
  Process::UID.change_privilege(original_uid)
end
```

Reply:
@Bo98 When you're back could do with your help figuring out how to do this better for `bundle install`.